custom_components/luxtronik/const.py

```diff
@@ -371,6 +372,17 @@ class LuxMkTypes(Enum):
         entity_category=EntityCategory.DIAGNOSTIC,
         sensor_key="calculations.ID_WEB_WMZ_Seit",
     ),
+        LuxtronikSensorEntityDescription(
+        key="current_heat_output",
+        # name=f"{text_heat_amount_counter}",
+        icon="mdi:lightning-bolt-circle",
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        unit_of_measurement=POWER_WATT,
+        #native_unit_of_measurement="W",
```

Comment:
I believe Home Assistant sensors are supposed to only use `native_unit_of_measurement` these days.